policies/tenant_cluster.json

```diff
@@ -56,57 +56,35 @@
                 "iam:UpdateAssumeRolePolicy",
                 "iam:UpdateRoleDescription"
             ],
-            "Resource": "arn:aws:iam::<CUSTOMER_ACCOUNT_ID>:role/*-EC2-K8S-Role"
+            "Resource": [
+                "arn:aws:iam::<CUSTOMER_ACCOUNT_ID>:role/*-EC2-K8S-Role",
+                "arn:aws:iam::<CUSTOMER_ACCOUNT_ID>:role/*-Route53Manager-Role",
+                "arn:aws:iam::<CUSTOMER_ACCOUNT_ID>:role/gs-*",
```

Comment:
I understand that this is the desired rule we want to establish now?

Reply:
No, this is needed for the current state. When fmt PR is approved it will be the source for the transition. There is a ticket in our roadmap assigned to firecracker team. The ticket description suggests how can be done the transition to the new naming.

But with the latest node pools, we need to add these changes in order to let aws operator create all resources